src/middleware/authorisation.coffee

```diff
@@ -85,8 +90,8 @@ exports.authorise = (ctx, done) ->
               # deny access to channel if the content type isn't set
               continue
 
-          # now check if message content matches
-          if matchContent(channel, ctx.body) is true
+          # now check if message content matches and that the status is 'enabled'
+          if matchContent(channel, ctx.body) and isChannelEnabled channel
```

Comment:
Perhaps we should check if a channel is enabled first rather than doing all this (possibly expensive) processing only to find out a channel is disabled?